samples/XAMLCatalog/XAMLCatalog/UIButtonViewController.m

```diff
@@ -278,8 +278,26 @@ - (UITableViewCell*)tableView:(UITableView*)tableView cellForRowAtIndexPath:(NSI
 
         [cell addSubview:label];
         [cell addSubview:button];
+    } else if (indexPath.row == 14) {
+        UILabel* label = [[UILabel alloc] initWithFrame:CGRectMake(_marginLeft, _marginTop, _labelWidth, _labelHeight)];
+        label.text = @"UIButton, Selected=true, Title=Blue";
+        
+        UIButton* button =
+        [[UIButton alloc] initWithFrame:CGRectMake(_marginLeft, _marginTop + _labelHeight, _defaultWidth, _defaultHeight)];
+        button.selected = true;
```

Comment:
With selected == true, do we ever encounter the normal state? How do we move from selected to normal and vice-versa.